ivy/functional/backends/paddle/activations.py

```diff
@@ -84,8 +84,12 @@ def gelu(
     return F.gelu(x, approximate=approximate)
 
 
+@with_unsupported_device_and_dtypes(
+    {"2.5.1 and below": {"cpu": ("bfloat16", "complex128", "complex64")}},
+    backend_version,
+)
```

Comment:
It looks to me like this function already has a section for handling complex numbers, which means they should probably be supported. Does that code not work?

Reply:
To clarify a bit, adding complex types to `with_unsupported_dtypes` to make tests pass should be a last resort during this process, as we're trying to *add* support for complex types

Reply:
The error it shows is `Selected wrong DataType `complex64`. Paddle support following DataTypes: float64, float32.` which obviously means that paddle doesn't support it.